File: src/main/java/no/nav/veilarbaktivitet/aktivitet/dto/EksternAktivitetDTO.java
package no.nav.veilarbaktivitet.aktivitet.dto;

import no.nav.veilarbaktivitet.aktivitetskort.*;
import no.nav.veilarbaktivitet.aktivitetskort.dto.Attributt;
import no.nav.veilarbaktivitet.aktivitetskort.dto.Etikett;
import no.nav.veilarbaktivitet.aktivitetskort.dto.LenkeSeksjon;
import no.nav.veilarbaktivitet.aktivitetskort.dto.Oppgaver;

import java.util.List;

public record EksternAktivitetDTO(
    AktivitetskortType type,
    Oppgaver oppgave,
    List<LenkeSeksjon> handlinger,
    List<Attributt> detaljer,
    List<Etikett> etiketter
) {
}

File: src/main/kotlin/no/nav/veilarbaktivitet/aktivitet/dto/EksternAktivitetDTO.kt
package no.nav.veilarbaktivitet.aktivitet.dto

import no.nav.veilarbaktivitet.aktivitetskort.dto.AktivitetskortType
import no.nav.veilarbaktivitet.aktivitetskort.dto.aktivitetskort.Attributt
import no.nav.veilarbaktivitet.aktivitetskort.dto.aktivitetskort.Etikett
import no.nav.veilarbaktivitet.aktivitetskort.dto.aktivitetskort.LenkeSeksjon
import no.nav.veilarbaktivitet.aktivitetskort.dto.aktivitetskort.Oppgaver

data class EksternAktivitetDTO(
    val type: AktivitetskortType,
    val oppgave: Oppgaver?,
    val handlinger: List<LenkeSeksjon>?,
    val detaljer: List<Attributt>?,
    val etiketter: List<Etikett>?
)
